janusgraph-core/src/main/java/org/janusgraph/core/attribute/Geoshape.java

```diff
@@ -16,22 +16,38 @@
 
 import com.google.common.base.Preconditions;
 import com.google.common.primitives.Doubles;
-import com.spatial4j.core.context.SpatialContext;
+import com.spatial4j.core.context.SpatialContextFactory;
+import com.spatial4j.core.context.jts.DatelineRule;
+import com.spatial4j.core.context.jts.JtsSpatialContext;
+import com.spatial4j.core.context.jts.JtsSpatialContextFactory;
 import com.spatial4j.core.distance.DistanceUtils;
+import com.spatial4j.core.io.jts.JtsBinaryCodec;
+import com.spatial4j.core.io.jts.JtsGeoJSONReader;
+import com.spatial4j.core.io.jts.JtsGeoJSONWriter;
+import com.spatial4j.core.io.jts.JtsWKTReader;
+import com.spatial4j.core.io.jts.JtsWKTWriter;
+import com.spatial4j.core.shape.Circle;
+import com.spatial4j.core.shape.Rectangle;
 import com.spatial4j.core.shape.Shape;
 import com.spatial4j.core.shape.SpatialRelation;
 import org.janusgraph.diskstorage.ScanBuffer;
 import org.janusgraph.diskstorage.WriteBuffer;
 import org.janusgraph.graphdb.database.idhandling.VariableLong;
-import org.janusgraph.graphdb.relations.RelationIdentifier;
-import org.apache.commons.lang.builder.HashCodeBuilder;
+
+import com.spatial4j.core.shape.jts.JtsGeometry;
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.Geometry;
+import com.vividsolutions.jts.geom.GeometryFactory;
```

Comment:
@pluradj Put another way, I think removing JTS dependency in this PR is way out of scope for this PR. Titan/JanusGraph depended on JTS up until today, so I think it is OK to upgrade these versions and then figure out a way off of current JTS. Sound OK?